### ISSUE TITLE
possible bug: I could not find model.eval() in deeprank

### ISSUE THREAD

**Issue by LilySnow** [DeepRank/deeprank#126]
I could not find `model.eval()`, which is required when we do evaluation especially when we used batchnorm in training. Did I miss something here?

**Comment by CunliangGeng** [comment]
It's OK, we used `train(False)` which is an alternative of `model.eval`, see https://pytorch.org/docs/stable/nn.html#torch.nn.Module.eval.
